sdk/identity/identity/review/identity.api.md

```diff
@@ -287,6 +288,17 @@ export interface ErrorResponse {
     traceId?: string;
 }
 
+// @public
+export function getBearerTokenProvider(credential: TokenCredential, scopes: string | string[], options?: GetBearerTokenProviderOptions): () => Promise<string>;
```

Comment:
does this mean we'd write a library that has a direct dependency on `@azure/identity`? Or would we expect the consumer to call this and pass in the function delegate?

Reply:
Exactly the latter. The customer experience would look along the lines of:
```ts
import { getBearerTokenProvider, DefaultAzureCredential } from "@azure/identity";
...
const credential = new DefaultAzureCredential();
const scope = "https://cognitiveservices.azure.com/.default";
const azureAdTokenProvider = getBearerTokenProvider(credential, scope);
...
const client = new ExternalClient({ ..., azureAdTokenProvider })
```

Reply:
This is meant to be used with clients in external packages that don't depend on our core, which in this case is https://www.npmjs.com/package/openai.

Reply:
oh _that_ openai 😄 SGTM